lib/ws_servers/api/algos/algo_worker.js

```diff
@@ -132,9 +168,12 @@ class AlgoWorker {
     d('issued reconnect [dms %s]', dms)
   }
 
-  registerEvents () {
+  /**
+   * @private
+   */
+  _registerEvents () {
```

Comment:
are we sure that these renames won't cause any issues anywhere else?

Reply:
Let's focus on the fix for now, I'll postpone those cosmetic changes to another PR.